cmd/nerdctl/run.go

```diff
@@ -55,6 +56,11 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	nerdctlInitBinary     = "nerdctl-init"
+	nerdctlInitBinaryPath = "/bin/" + nerdctlInitBinary
```

Comment:
Why not exec.LookPath?

Reply:
This is the path which used in container